core/src/test/java/com/microsoft/applicationinsights/telemetry/BaseTelemetryTest.java

```diff
@@ -23,11 +23,17 @@
 
 import com.microsoft.applicationinsights.internal.schemav2.Data;
 import com.microsoft.applicationinsights.internal.schemav2.Domain;
+import com.microsoft.applicationinsights.internal.schemav2.Envelope;
+import com.microsoft.applicationinsights.internal.schemav2.SessionStateData;
+
 import org.junit.Test;
 
 import java.io.IOException;
+import java.io.StringWriter;
+import java.lang.reflect.*;
```

Comment:
@XiaoliangHuang can we avoid use of wild cards for import here?

Reply:
this was added as I planed to use reflect to test the private method, but at last not used, so I removed it.